### PR TITLE
add downgrade example to whatsnew

### PR DESCRIPTION
**Pull request by minrk**
and discuss `--to notebook` in nbconvert doc

### CODE REVIEW

File: docs/source/notebook/nbconvert.rst [hunk 1]
```diff
@@ -78,18 +78,44 @@ The currently supported export formats are:
...
+  
+      ipython nbconvert --to notebook --nbformat 3 mynotebook.ipynb
+  
+  will create a copy of :file:`mynotebook.ipynb` in :file:`mynotebook.v3.ipynb`
+  in version 3 of the :ref:`notebook format <nbformat>`.
+  
+  If you want to convert a notebook in-place (careful!),
+  you can specify the ouptut file to be the same as the input file::
+  
+      ipython nbconvert --to notebook mynb --output mynb ...
```
Comment (Carreau) [inline docs/source/notebook/nbconvert.rst]: dot dot dot ? I would also be consistent on wether or not you add the .ipynb at the end of the filenames.

File: docs/source/whatsnew/version3.rst [hunk 2]
```diff
@@ -217,6 +217,12 @@ Backwards incompatible changes
     to read and write notebook files
     instead of the deprecated :mod:`IPython.nbformat.current` APIs.
 
+  You can downgrade a notebook to v3 via ``nbconvert``::
+  
+      ipython nbconvert --to notebook --nbformat 3 [notebook]
```
Comment (Carreau) [inline docs/source/whatsnew/version3.rst]: `<notebook>` ? isn't `[]` for optional and `<>` for required usually ? That's my feeling, but I'm not sure.

File: docs/source/notebook/nbconvert.rst [hunk 3]
```diff
@@ -78,18 +78,44 @@ The currently supported export formats are:
...
+  :file:`mynotebook.nbconvert.ipynb`.
+  
+  ::
+  
+      ipython nbconvert --to notebook --nbformat 3 mynotebook.ipynb
+  
+  will create a copy of :file:`mynotebook.ipynb` in :file:`mynotebook.v3.ipynb`
+  in version 3 of the :ref:`notebook format <nbformat>`.
+  
+  If you want to convert a notebook in-place (careful!),
```
Comment (takluyver) [inline docs/source/notebook/nbconvert.rst]: I don't think the 'careful!' here is needed, since you have a warning after the example.

**Comment by minrk** [comment]
comments should be addressed, thanks.

**Pull request merged**